base_layer/core/src/base_node/comms_interface/inbound_handlers.rs

```diff
@@ -374,6 +374,8 @@ where B: BlockchainBackend + 'static
                         .await?,
                     self.consensus_manager.get_block_reward_at(height),
                 );
+
+                debug!(target: LOG_TARGET, "New template block: {}", block_template);
```

Comment:
```suggestion
                trace!(target: LOG_TARGET, "New template block: {}", block_template);
```
this can be a very large log

Reply:
Fair enough, I'll leave that for another PR so that we dont have to rerun tests